### PR TITLE
Add and test x86_64 emulator support on s390x and aarch64

### LINKED ISSUE #1088: New Package Request: qemu-user-static
Please try to answer the following questions about the package you are requesting:

1. What, if any, are the additional dependencies on the package? (i.e. does it pull in Python, Perl, etc)

qemu

2. What is the size of the package and its dependencies?

Not sure

3. What problem are you trying to solve with this package? Or what functionality does the package provide?

This package allows running x86 programs on aarch64 for docker and podman. The use case is in podman's vm for Mac: https://github.com/containers/podman/discussions/12899#discussioncomment-2068854

4. Can the software provided by the package be run from a container?  Explain why or why not.

No. This software allows for running containers.

5. Can the tool(s) provided by the package be helpful in debugging container runtime issues?

Not likely.

6. Can the tool(s) provided by the package be helpful in debugging networking issues?

Not likely.

7. Is it possible to layer the package onto the base OS as a day 2 operation?  Explain why or why not.

Not sure

8. In the case of packages providing services and binaries, can the packaging be adjusted to just deliver binaries?

This is just binaries.

9. Can the tool(s) provided by the package be used to do things we’d rather users not be able to do in FCOS? (e.g. can it be abused as a Turing complete interpreter?)

Not sure

10. Does the software provided by the package have a history of CVEs?

Not sure. CVEs for QEMU itself are here: https://www.cvedetails.com/vulnerability-list/vendor_id-7506/Qemu.html

### PR DESCRIPTION
**Pull request by gursewak1997**
Fixes: [https://github.com/coreos/fedora-coreos-tracker/issues/1237](https://github.com/coreos/fedora-coreos-tracker/issues/1088)

### CODE REVIEW

File: tests/kola/binfmt/qemu [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# kola: {"exclusive": false, "tags": "needs-internet"}
+# https://github.com/coreos/fedora-coreos-tracker/issues/1237
+# Test the x86_64 emulator on aarch64 and s390x machines for now
```
Comment (dustymabe) [inline tests/kola/binfmt/qemu]: Let's add rationale for each kola.json entry like https://github.com/coreos/fedora-coreos-config/blob/c048af0980da12cf6d339f5d8b27e62eb1259c4d/tests/kola/podman/dns/test.sh#L4-L18

We can probably limit this test to qemu (i.e. if it works anywhere it should work anywhere).

Also I wonder if we should limit this here to just s390x and aarch64 with kola.json entries for `architectures`, then we'd made the `*)` in the case statement be `fatal` (as in should not get there).

File: manifests/fedora-coreos-base.yaml [hunk 2]
```diff
@@ -204,6 +204,9 @@ packages-ppc64le:
   - ppc64-diag-rtas
 packages-aarch64:
   - irqbalance
+  - qemu-user-static-x86
+packages-s390x:
+  - qemu-user-static-x86
```
Comment (dustymabe) [inline manifests/fedora-coreos-base.yaml:213]: let's add a linebreak to the comment block above and add a new comment explaining why we 're adding these.

**Review by dustymabe — COMMENTED** [review]
Code changes mostly look good. A few suggestions.

File: tests/kola/binfmt/qemu (hunk 1 above)
Comment (gursewak1997) [reply tests/kola/binfmt/qemu]: > Also I wonder if we should limit this here to just s390x and aarch64 with kola.json entries for `architectures`, then we'd made the `*)` in the case statement be `fatal` (as in should not get there).

I initially added that but thought we might have new changes for other architectures soon. Nonetheless, I'll add the `architectures` entry to kola.json

File: manifests/fedora-coreos-base.yaml [hunk 3]
```diff
@@ -195,6 +195,10 @@ packages:
 # Anyways, it was requested by the Red Hat perf team for RHCOS, so we have it here.
 # https://serverfault.com/questions/513807/is-there-still-a-use-for-irqbalance-on-modern-hardware
 # https://access.redhat.com/solutions/41535
+# Include the qemu-user-static-x86 package on aarch64 and s390x FCOS images
+# to allow access to the large inventory of containers only built for x86_64.
+# https://github.com/coreos/fedora-coreos-tracker/issues/1237
+
```
Comment (dustymabe) [inline manifests/fedora-coreos-base.yaml]: optional: move line break to break up comments

```suggestion
# https://access.redhat.com/solutions/41535
#
# Include the qemu-user-static-x86 package on aarch64 and s390x FCOS images
# to allow access to the large inventory of containers only built for x86_64.
# https://github.com/coreos/fedora-coreos-tracker/issues/1237
```

**Review by dustymabe — APPROVED** [review]
LGTM

**Review by dustymabe — CHANGES_REQUESTED** [review]
Actually we need to fix the commit message.. We should be linking to/fixing https://github.com/coreos/fedora-coreos-tracker/issues/1088

**Review by dustymabe — APPROVED** [review]
LGTM

**Pull request merged**